pkg/netflow/payload/payload.go

```diff
@@ -44,7 +44,7 @@ type FlowPayload struct {
 	Packets      uint64           `json:"packets"`
 	EtherType    string           `json:"ether_type,omitempty"`
 	IPProtocol   string           `json:"ip_protocol"`
-	Exporter     Exporter         `json:"exporter"`
+	Device       Device           `json:"device"`
```

Comment:
Note for reviewers: this is the actual change. The other changes are just variable name renaming.